### PR TITLE
fix(analytics): update startingOptions to reflect schema

### PR DESCRIPTION
**Pull request by Yelinz**
## Description

Add `DOCUMENTS` and `WORK_ITEMS` startingOptions to reflect the schema

### CODE REVIEW

File: packages/analytics/addon/components/ca-report-builder.js [hunk 1]
```diff
@@ -12,9 +12,12 @@ export default class CaReportBuilderComponent extends Component {
...
 
   get startingObjects() {
-    // TODO: Replace with dynamic list
-    // return this.args.startingObjects;
-    return [{ label: "Cases", value: "CASES" }];
+    // startingObjects defined by schema
+    return [
+      { label: "Cases", value: "CASES" },
+      { label: "Work items", value: "WORK_ITEMS" },
+      { label: "Documents", value: "DOCUMENTS" },
```
Comment (anehx) [inline packages/analytics/addon/components/ca-report-builder.js:19]: This needs to be translated

**Review by anehx — CHANGES_REQUESTED** [review]

File: packages/analytics/addon/components/ca-report-builder.js (hunk 1 above)
Comment (Yelinz) [reply packages/analytics/addon/components/ca-report-builder.js]: added translations

File: packages/analytics/translations/de.yaml [hunk 2]
```diff
@@ -42,3 +42,7 @@ caluma:
       filter-placeholder: Filter...
       delete-filter: Filter löschen
       empty: Keine erfassten Filter
+    starting-options:
+      cases: Fälle
```
Comment (anehx) [inline packages/analytics/translations/de.yaml:46]: Let's use "Dossiers"

**Review by anehx — CHANGES_REQUESTED** [review]

File: packages/analytics/translations/de.yaml (hunk 2 above)
Comment (Yelinz) [reply packages/analytics/translations/de.yaml]: Done

**Review by anehx — APPROVED** [review]

**Pull request merged**